### DISCUSSION TITLE
Inquiry 5

### DISCUSSION THREAD

**Discussion by yujispin990** [winstxnhdw/lc-hax#74] [Q&A] [answered]
### What happened?

a description for demigod please.

### Current Commit Hash

ffb9c81

### Injector

- [ ] SharpMonoInjectorCore
- [ ] Others

### Acknowledgement

- [X] I have **recursively** cloned the repository as instructed in the [README](https://github.com/winstxnhdw/lc-hax/edit/main/README.md)
- [X] I have done my due diligence to check for similar issues

**Comment by winstxnhdw** [comment, marked as answer]
It's just a weaker variant of `/god` but you can attack your teammates.

**Comment by yujispin990** [comment]
gotcha!

**Comment by yujispin990** [comment]
also do you have some sort of a discord channel where we could discuss about these things?

**Comment by winstxnhdw** [comment]
No, but we can use the [Discussions](https://github.com/winstxnhdw/lc-hax/discussions) tab on GitHub.

**Comment by yujispin990** [comment]
ah nice! i shall use that then.
